src/protocolsupport/protocol/typeremapper/pe/PEDataValues.java

```diff
@@ -140,7 +141,12 @@ private static void registerLivingType(NetworkEntityType type, int networkId) {
 	}
 
 	public static int getEntityTypeId(NetworkEntityType type) {
-		return entityType.get(type);
+		try {
+			return entityType.get(type);
+		} catch(NullPointerException e) {
```

Comment:
How about a space before (? :-)

Reply:
woops, still getting used to this code style

Reply:
"This" style?! How on earth can you code without spaces after keywords?! :-o </religious_rant> ;-)

Reply:
lol i know, im just evil that way